libs/language-server/src/lib/validation/checks/blocktype-specific/property-assignment.ts

```diff
@@ -264,7 +264,9 @@ function checkLocalFileExtractorProperty(
   validationContext: ValidationContext,
 ) {
   if (propName === 'filePath') {
-    if (typeof propValue === 'string' && propValue.startsWith('../')) {
+    const validPathRegex = /^(?!.*\.\.\/).*$/;
```

Comment:
It would be much easier to test for an invalid path (which is anything that does include two dots I assume but please google a bit for if that makes sense). So I'd invert this check and check for invalid paths and if that is true, throw the error.